testsuite/tests/typing-modules/functors.ml

```diff
@@ -221,7 +221,7 @@ module Ord : sig type t = unit val compare : 'a -> 'b -> int end
 Line 2, characters 11-29:
 2 | module M = Map.Make(Ord)(Ord)
                ^^^^^^^^^^^^^^^^^^
-Error: The functor application is ill-typed.
+Error: This functor application of Map.Make is ill-typed.
```

Comment:
I'm not fond of the wording "This functor application of Map.Make". Alternative proposal: "The application of the functor Map.Make".